portfolio/src/main/webapp/index.html

```diff
@@ -41,7 +42,7 @@ <h1>Zoe Mitchell</h1>
                          during stressful situations, improve my self-worth and confidence. In the words of Winnie 
                          the Pooh, 
                          "a friend is one of the nicest things you can have and one of the best things you can be". 
-                        </p1>
```

Comment:
Did you mean to keep the second `</p1>`? It looks like there are two closing `</p1>`s for the first `<p1>`